extensions/sql-migration/README.md

```diff
@@ -1,49 +1,53 @@
+- Your database backup location details, either a network file share or an Azure Blob Storage container.
 
 ## Getting started
 Refer to [Migrate databases using the Azure SQL Migration extension for Azure Data Studio](https://docs.microsoft.com/azure/dms/migration-using-azure-data-studio) for detailed documentation on capabilities and concepts.
 
 ## Assessment and SKU recommendation
-The assessment and SKU recommendation feature
-- evaluates the source SQL Server database(s) for migration readiness.
-- generates right-sized SKU recommendations in Azure to meet the performance requirements of the source database(s) with minimal cost. [Learn more.](https://aka.ms/ads-sql-sku-recommend)
+The assessment and SKU recommendation feature evaluates the source SQL Server database(s) for migration readiness.
+
+It also generates right-sized SKU recommendations for your Azure target to meet the performance requirements of the source SQL Server database(s) with minimal cost. [Learn more.](https://aka.ms/ads-sql-sku-recommend)
 
 ## Azure SQL targets
 The Azure SQL Migration extension supports database migrations to the following Azure SQL targets.
-- [SQL on Azure Virtual Machines (Windows)](https://docs.microsoft.com/azure/azure-sql/virtual-machines/windows/sql-server-on-azure-vm-iaas-what-is-overview)
 - [Azure SQL Managed Instance](https://docs.microsoft.com/azure/azure-sql/managed-instance/sql-managed-instance-paas-overview)
+- [SQL Server on Azure Virtual Machines](https://docs.microsoft.com/azure/azure-sql/virtual-machines/windows/sql-server-on-azure-vm-iaas-what-is-overview)
+- [Azure SQL Database](https://docs.microsoft.com/en-us/azure/azure-sql/database/sql-database-paas-overview?view=azuresql)
 
 
 ## Migration modes
 The following migration modes are supported for the corresponding Azure SQL targets.
-- Online - The source SQL Server database is available for read and write activity while database backups are continuously restored on target Azure SQL. Application downtime is limited to duration for the cutover at the end of migration.
-- Offline - The source database cannot be used for write activity while database backup files are restored on the target Azure SQL database. Application downtime persists through the start until the completion of the migration process.
+- Online - The source SQL Server database is available for read and write activity, while the database backups (full + log) are continuously restored on the Azure SQL target. Application downtime is limited to the duration of the cutover at the end of migration.
+    - Azure SQL Database is not yet supported for online migrations.
+- Offline - The source SQL Server database cannot be used for write activity, while the database backup files are restored on the Azure SQL target. Application downtime persists from the start until the completion of the migration process.
```

Comment:
can we specify manual cutover here

Reply:
Thanks for the feedback @rohitdhamankar. As discussed offline, I will take a look at the documentation and add more details as required. Let's keep the readme file simple.